### PR TITLE
Support wget for download, not just curl

### PR DESCRIPTION
**Pull request by justinsb**
Some images don't include curl but do have wget.  Prefer curl, but
fallback to wget.

(The official Debian 9 image does not have curl.)

### CODE REVIEW

File: pkg/model/resources/nodeup.go [hunk 1]
```diff
@@ -66,8 +66,13 @@ download-or-bust() {
     for url in "${urls[@]}"; do
       local file="${url##*/}"
       rm -f "${file}"
-      if ! curl -f --ipv4 -Lo "${file}" --connect-timeout 20 --retry 6 --retry-delay 10 "${url}"; then
-        echo "== Failed to download ${url}. Retrying. =="
+
+      if [[ $(which curl) ]]; then
```
Comment (chrislovecnm) [inline pkg/model/resources/nodeup.go]: just a note.  We need to allow for s3cmd and gs :)

**Comment by chrislovecnm** [comment]
CI is not passing

File: pkg/model/resources/nodeup.go (hunk 1 above)
Comment (justinsb) [reply pkg/model/resources/nodeup.go]: I think both can do signed URLs.

**Comment by justinsb** [comment]
Fixed CI :-)

**Pull request merged**